server.js

```diff
@@ -204,8 +220,8 @@ app.use((req, res)=>{
 	const props = {
 		version     : require('./package.json').version,
 		url         : req.originalUrl,
-		welcomeText : welcomeText,
-		changelog   : changelogText,
+		welcomeText : req.welcomeText,
+		changelog   : req.changelogText,
```

Comment:
Then if everything is contained in the brews both of these lines can just be deleted.

Reply:
And in Homebrew.jsx we can delete them from the initialProps.

Reply:
Deleted in the latest commit; confirmed that both the home page and change log page still work correctly.